custom_model_runner/datarobot_drum/drum/gpu_predictors/nemo_predictor.py

```diff
@@ -94,6 +113,54 @@ def supported_payload_formats(self):
     def has_read_input_data_hook(self):
         return False
 
+    def _get_custom_artifacts(self):
+        code_dir_abspath = os.path.abspath(self._code_dir)
+
+        custom_py_paths = list(Path(code_dir_abspath).rglob("{}.py".format(CUSTOM_FILE_NAME)))
+        dot_remote_paths = list(Path(code_dir_abspath).rglob(DOT_REMOTE))
+
+        if len(custom_py_paths) + len(dot_remote_paths) > 1:
+            error_mes = (
+                "Multiple custom.py/.remote files were identified in the code directories sub directories.\n"
+                "The following custom model files were found:\n"
+            )
+            error_mes += "\n".join([str(path) for path in (custom_py_paths + dot_remote_paths)])
+            self.logger.error(error_mes)
+            raise DrumCommonException(error_mes)
+
+        return custom_py_paths, dot_remote_paths
+
+    def _load_model_remote_artifact(self):
+        from datarobot_custom_code.model_artifact_downloader import ModelDownloader
+
+        deployment_id = os.environ["MLOPS_DEPLOYMENT_ID"]
```

Comment:
This won't exist during custom models tests. Won't this break?

Reply:
correct, it should not be in the CM tests. I'll update it

Reply:
Removed this library at all. I'll create a separate tickets to make:
- tests
- harness pipeline
- fix logging and other misc things